src/main/java/roomescape/Service/ReservationService.java

```diff
@@ -0,0 +1,29 @@
+package roomescape.Service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import roomescape.DTO.ReservationDTO;
+import roomescape.Domain.Reservation;
+import roomescape.Domain.Time;
+import roomescape.Repository.ReservationRepository;
+
+import java.util.List;
+
+@Service
+public class ReservationService {
+    @Autowired
+    private ReservationRepository reservationRepository;
+    public List<Reservation> allReservation(){
+        return reservationRepository.getAllReservation();
+    }
+
+    public Reservation createdReservation(ReservationDTO reservation){
```

Comment:
DTO를 사용한 점 좋아요!! DTO를 사용하는 목적을 알아보는 것도 좋답니당😄 
이 메소드에서는 ID에 대한 값을 제외한 name, date, time을 받는 DTO이고, 예약 생성에 관한 DTO이므로 `ReservationCreateRequestDTO`같은 이름을 짓는 것도 좋을 거 같네요! (개행..!)

```suggestion
    public Reservation createdReservation(ReservationCreateRequestDTO reservationCreateRequest) {
```